packages/react-components/src/components/Datepicker/helpers.ts

```diff
@@ -0,0 +1,144 @@
+import {
+  isAfter,
+  isSameDay,
+  isSameMonth,
+  subMonths,
+  differenceInCalendarDays,
+} from 'date-fns';
+import { Modifiers } from 'react-day-picker';
+import {
+  IRangeDatePickerProps,
+  IRangeDatePickerState,
+  IRangeDatePickerOption,
+} from './types';
+
+const baseClass = 'date-picker';
```

Comment:
```suggestion
const baseClass = 'lc-date-picker';
```

?

Reply:
We have dropped this convention along the way.